Livecheckables/spidermonkey.rb

```diff
@@ -0,0 +1,6 @@
+class Spidermonkey
+  livecheck do
+    url "https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Releases"
+    regex(%r{href=.*?Releases/v?(\d+(?:\.\d+)*)}i)
```

Comment:
```suggestion
    regex(%r{href=.*?Releases/v?(\d+(?:\.\d+)*)[ '">]}i)
```

This could use a delimiter, to deal with the very unlikely case that they post an unstable release at some point.